fe/fe-core/src/main/java/org/apache/doris/common/util/VectorizedUtil.java

```diff
@@ -20,18 +20,9 @@
 import org.apache.doris.qe.ConnectContext;
 
 public class VectorizedUtil {
```

Comment:
Why not just remove this file?

Reply:
because many code depend on isVectorized Method, too many modifications and may remove some if else clause.  I am afraid some mistakes. So that I just return true here and may modify these code later.